# Example of exception
T = int(input("Enter an integer: "))
print("The number you enter is ", T)
print("Bye!")